docs/guides/qiskit-code-assistant-vscode.mdx

```diff
@@ -1,24 +1,24 @@
 ---
-title: Use the Qiskit Code Assistant in VSCode
-description: How to install, use, and configure the Qiskit Code Assistant in Visual Studio Code (VSCode).
+title: Use Qiskit Code Assistant in VS Code
+description: How to install, use, and configure Qiskit Code Assistant in Visual Studio Code (VS Code).
 ---
 
-# Use the Qiskit Code Assistant in Visual Studio Code
+# Use Qiskit Code Assistant in Visual Studio Code
 
-Learn how to install, use, configure and uninstall the official Qiskit Code Assistant extension in Visual Studio Code (VSCode).
+Learn how to install, use, configure and uninstall the official Qiskit Code Assistant extension in Visual Studio Code (VS Code).
```

Comment:
weird - I thought you already updated all of these to "VS Code" earlier!

Reply:
I did a search and replace and still missed this one 🤷